packages/next/lib/get-function-location.ts

```diff
@@ -0,0 +1,48 @@
+let id = 0
+
+export async function getFunctionLocation(fn: any): Promise<{
+  filename: string
+  line: number
+  column: number
+}> {
+  // use require to lazy load inspector
+  const { Session } = require('inspector') as typeof import('inspector')
+  const { promisify } = require('util')
+
+  id += 1
+  const session = new Session()
+  session.post = promisify(session.post) as any
```

Comment:
```suggestion
  const sessionPost = promisify(session.post);
```

IMHO we should not override the original `post` method. By extracting it to a variable can also enable the proper type check.